File: 프로그래머스/1/42748. K번째수/K번째수.cpp
#include <bits/stdc++.h>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    for (vector<int> com: commands) {
        int i = com[0] - 1;
        int j = com[1] - 1;
        vector<int> temp;
        for(int k = i; k <= j; k++) {
            temp.push_back(array[k]);
        }
        sort(temp.begin(), temp.end());
        answer.push_back(temp[com[2] - 1]);
    }
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.swift
import Foundation

func solution(_ array:[Int], _ commands:[[Int]]) -> [Int] {
    var res: [Int] = []
    for com in commands {
        let subArr = Array(array[(com[0] - 1)..<com[1]]).sorted()
        res.append(subArr[com[2] - 1])
    }
    return res
}